eq-author/src/App/QuestionnairesPage/QuestionnairesView/QuestionnairesTable/index.js

```diff
@@ -26,43 +30,131 @@ const TH = styled.th`
   font-size: 0.9em;
 `;
 
-TH.propTypes = {
-  colWidth: PropTypes.string.isRequired,
-};
+const SortButton = styled.span`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+  opacity: 0.8;
+
+  &::after {
+    content: url(${iconArrow});
+    display: inline-block;
+    width: 16px;
+    height: 16px;
+    opacity: ${props => (props.active ? "0.8" : "0.2")};
+    transform: rotate(${props => (props.order === ASC ? "180deg" : "0deg")});
+    transition: transform 150ms ease-out;
+  }
+
+  &:hover {
+    opacity: 1;
+  }
+`;
 
 const Panel = styled.div`
   background: white;
   border-radius: 4px;
   border: 1px solid #ccc;
 `;
 
+const SortableTH = ({
+  children,
+  sortOrder,
+  sortKey,
+  onSortClick,
+  onReverseClick,
+  colWidth,
+  currentSortKey,
+  dataTest,
+}) => {
+  const active = sortKey === currentSortKey;
+  return (
+    <TH aria-sort={active ? sortOrder : "none"} colWidth={colWidth}>
+      <SortButton
+        active={active}
+        order={active ? sortOrder : DESC}
+        role="button"
+        onClick={() => (active ? onReverseClick() : onSortClick(sortKey))}
+        data-test={dataTest}
+      >
+        {children}
+      </SortButton>
+    </TH>
+  );
+};
+
+const TableHead = props => {
```

Comment:
I think it might be worthwhile moving this to its own component.

Reply:
Moved this to its own component.